styleguide/components/SectionHeading/index.js

```diff
@@ -0,0 +1,34 @@
+import { css } from '@emotion/core';
+import React from 'react';
+import PropTypes from 'prop-types';
+import Heading from 'rsg-components/Heading';
+
+import theme from 'styleguide/theme';
+
+function SectionHeadingRenderer({ children, toolbar, id, href, depth, deprecated }) {
+	// const headingLevel = Math.min(6, depth);
+	// Overwrite RSG's classification of headings, since we're only using top-level headings here!
+	const headingLevel = 1;
+
+	return (
+			<Heading level={headingLevel} id={id}>
```

Comment:
Geeez what is happening in here with your indentation? 😨

Reply:
Probably it's copied from the RSG files directly on Github?

INSTALL A LINTER PLS.

Reply:
(Tell me if that's fixed it now? I can't easily tell!)